### PR TITLE
Add chpi to known channel list

### LINKED ISSUE #1323: ch_type_mapping has no entry for `chpi`
### Description of the problem

MEGIN systems store chpi information in a channel called 'CHPI00X', which is read by `mne` as ch_type `chpi`. However, this entry is missing in the mne to bids conversion dictionary.

This leads to 

```python
  File ~/anaconda3/lib/python3.11/site-packages/mne_bids/write.py:150 in _channels_tsv
    ch_type.append(map_chs[_channel_type])

KeyError: 'chpi'
```

### Steps to reproduce

Choose a raw file that contains cHPI channels

```Python
In [18]: mne.channel_type(raw.info, 330)
Out[18]: 'chpi'
```

then try to call `write_raw_bids` using that file or use the command line interface

```
write_raw_bids(raw=raw,  # raw containing CHPI MEGIN channels
		bids_path=bids_subj,
		overwrite=True,
		verbose=True)
```

### Solution

`chpi` should be added to the channel mapping dictionary, probably as `MEGOTHER` or `MISC`?

Somewhere after https://github.com/mne-tools/mne-bids/blob/a99fe4dbe0609c1206e8156965f384741658b743/mne_bids/utils.py#L62

### Additional information

I'm using `mne==1.8.0` and `mne_bids=='0.16.0.dev49+ga99fe4d'`

### PR DESCRIPTION
**Pull request by skjerns**
PR Description
--------------

cHPI was not in the mapping dictionary, I've added it and also added a test. 

resolves #1323

Additionally, I found that there are some other channels that are not known/converted and some channel types that are not known to MNE but are still in the mapping dictionary (can they in practice ever arise if they're not in MNE?)

I think we should be able to get an up to date channel list from this [code](https://github.com/mne-tools/mne-python/blob/56e522ba303fe01a25a2a0d5d7507639a83fe02c/mne/_fiff/pick.py#L154) and could use it in the test. However, the module is private, making it harder to access.

Merge checklist
---------------

Maintainer, please confirm the following before merging.
If applicable:

- [ ] All comments are resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/main/doc/whats_new.rst) is updated
- [ ] New contributors have been added to [CITATION.cff](https://github.com/mne-tools/mne-bids/blob/main/CITATION.cff)
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Comment by skjerns** [comment]
ready for review :)

**Review by sappelhoff — APPROVED** [review]
Thanks!

**Comment by sappelhoff** [comment]
Could you please add an entry to the changelog? Then we can merge it.

**Comment by skjerns** [comment]
Done!

edit: Apparently I was already part of the author's list - weird, I never contributed here. Maybe that's from my involvement of MNE?

File: CITATION.cff [hunk 1]
```diff
@@ -206,6 +206,10 @@ authors:
       family-names: Jas
       affiliation: 'Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Charlestown, MA, USA'
       orcid: 'https://orcid.org/0000-0002-3199-9027'
+    - given-names: Simon
+      family-names: Kern
+      affiliation: 'Central Institute of Mental Health, Mannheim, Germany'
+      orcid: 'https://orcid.org/0000-0002-9050-9040'
```
Comment (sappelhoff) [inline CITATION.cff]: You are already listed in this file :)


```suggestion
```

**Comment by sappelhoff** [comment]
you DID contribute in 2021! 😄 how the time flies:

- https://github.com/mne-tools/mne-bids/pull/912

**Comment by sappelhoff** [comment]
Thanks!

**Comment by skjerns** [comment]
> you DID contribute in 2021! 😄 how the time flies:
> 
> * [[MRG] fix loading without age information / single-column .tsv #912](https://github.com/mne-tools/mne-bids/pull/912)

haha, wow. I never actually used `mne-bids` after that :D

**Pull request merged**